### PR TITLE
Initial Daisy Seed Benchmarks for TIMBRE Net

### PR DESCRIPTION
**Pull request by Kaimul24**
Built and ran raw-C and TFlite Micro Daisy Seed inference benchmarks for the TIMBRE net. 

**Results**
Raw-C baseline of the TIMBRE net ran in roughly 220 ms with no quantization.

Not all PyTorch operations used by the model are supported by TFLite (InstanceNorm2D, GELU, stft/istft, etc); Had to separate the U-Net and Transient Shaper into separate Keras modules and manually create some of these operations. Kept the stft/istft in raw C. 

The quantized (int8) TFLite Micro model ran much slower (~2764 ms). This performance is likely from the custom, unoptimized operations (which are comprised of many suboperations), resulting in a bloated inference path. The quantization also showed a noticeable accuracy reduction compared to ground-truth floats.

### CODE REVIEW

**Review by sbarry753 — APPROVED** [review]
good work. even if we dont use this, good work
